README.md

```diff
@@ -138,115 +154,27 @@ Logging:
-of the requirements.
-
-Some of the potential API designs are:
-
-```yaml
-Packages:
-  - repository::package
-  - package@repository
-  - package
-    repo: repository
-```
-
-### Package dependencies:
-
-By default, packages will need Imports/Depends/LinkingTo to make
-sure the packages can work successfully. 
-
-```yaml
-Packages:
-  - PKPDmisc
-    Suggests: true
-  - dplyr
-```
-
-The benefit is customizations related to package requirements
-are immediately visible. The downside is it "pollutes" the
-packages list.
-
-```yaml
-Packages:
-  - PKPDmisc
-  - dplyr
-
-Customizations:
-  PKPDmisc:
-    Suggests: true
-```
-
-
-## Assumptions (for now)
-
-Making this tool bulletproof will take a significant effort over time. To bring confidence for use day-to-day
-we must clearly outline the assumptions we are making internally to provide guidance on the areas
-this tool may not be suitable for, or to explain unexpected behavior.
-
-* Package/versions from a given repo will not change over time
-  * if pkgx_0.1.0 is downloaded from repoY, we do not need to check each time that pkgx is consistent
-  * this allows simple caching without doing hash comparisons (for now)
-
-R package management
-
-## Install Strategy Background
-
-One of the problems with the full layered implementation is the longest install time dictates the entire layer
-installation install. Originally, we did not know if this would be a huge problem, however it was quickly
-evident that this was not the case.
-
-For example, when look at the installation layers given a request for ggplot2, the following was
-the installation timing. For layer 1, the second longest install time was Rcpp (39 seconds), with most
-other packages coming in less than 10 seconds.
-
-| layer  |package | duration|
-|:-------|:-------|--------:|
-|   1    |stringi |   159.39|
-|   2    |Matrix  |    69.98|
-|   3    |mgcv    |    34.45|
-|   4    |tibble  |     2.37|
-|   5    |ggplot2 |    12.12|
-
-Furthermore, when looking at subsequent layers, neither Matrix or mgcv and its dependencies have any relation
-to stringi, so there is no reason to wait for the layer to complete.
-
-# Development
-
-run all tests with tabular output: 
-
-```
-go test ./... -json -cover | tparse -all 
-```
\ No newline at end of file
+## Pkgr and [Packrat](https://rstudio.github.io/packrat/)
+
+**Pkgr is not a replacement for Packrat**. Packrat is a tool to capture the state
+of your R environment and isolate it from outside modification.
+Where Packrat often falls short, however, is in the restoration said environment.
+Packrat::restore() restores packages in an iterative fashion, which is a
+time-consuming process that doesn't always play nice with packages hosted outside
+of CRAN (such as packages hosted on GitHub). Additionally, packrat::restore()
+provides the user with almost no control of the _order_ in which packages are
+restored -- it always installs packages alphabetically by name.
```

Comment:
it still has to install them with dependencies first. pkgr doesn't let the user control load order either.

A more relevant problem is, packrat, which uses install.packages under the hood, runs into the same problems that arise with install.packages, namely, a package installation failing will not cause the system to stop with an informative error, rather it will continue to attempt to install additional packages and not communicate the errors in a proper and timely fashion